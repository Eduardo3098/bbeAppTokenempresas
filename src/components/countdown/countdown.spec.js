import {shallow} from 'enzyme';
import React from 'react';
import Countdown from './countdown';

describe('Countdown widget', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Countdown />);
  });

  test('renders correctly', () => {
    expect(wrapper.find({testID: 'component'})).toHaveLength(1);
  });
});
