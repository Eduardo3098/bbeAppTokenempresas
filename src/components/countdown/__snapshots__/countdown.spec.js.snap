// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Origin screen tests Should render correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#7e7e7e",
      },
      undefined,
    ]
  }
>
  0:0:01:39
</Text>
`;
